addons/shootingrange/stringtable.xml

```diff
@@ -3,168 +3,219 @@
+            <German>Liste der möglichen Schießstand Laufzeiten in Sekunden. Durch Kommas trennen. Wird nur im Zeit- und Treffermodus (Zeit begrenzt) genutzt. Standard: 0, 30, 60, 150, 300</German>
         </Key>
         <Key ID="STR_TAC_ShootingRange_DefaultDuration">
             <English>Default Duration</English>
+            <German>Standardzeit</German>
         </Key>
         <Key ID="STR_TAC_ShootingRange_DefaultDurationDesc">
             <English>Default shooting duration in seconds. Only used in Time and Hit (Time Limited) Modes. Default: 60</English>
+            <German>Standardzeit in Sekunden. Wird nur im Zeit- und Treffermodus (Zeit begrenzt) genutzt.</German>
         </Key>
         <Key ID="STR_TAC_ShootingRange_TargetAmounts">
             <English>Target Amounts</English>
+            <German>Anzahl der Ziele</German>
         </Key>
         <Key ID="STR_TAC_ShootingRange_TargetAmountsDesc">
             <English>List of available number of target pop-ups, separated by commas. Only used in Hit (Target Limited) Mode. Default: 5, 10, 15, 30, 45</English>
+            <German>Liste der möglichen Zahl aufklappbarer Ziele. Durch Kommas trennen. Wird nur im Treffermodus (Ziele begrenzt) genutzt. Standard: 15</German>
         </Key>
         <Key ID="STR_TAC_ShootingRange_DefaultTargetAmount">
             <English>Default Target Amount</English>
+            <German>Standard Anzahl an Zielen</German>
         </Key>
         <Key ID="STR_TAC_ShootingRange_DefaultTargetAmountDesc">
             <English>Default amount of target pop-ups. Only used in Hit (Target Limited) Mode. Default: 15</English>
+            <German>Standard Anzahl an aufklappbaren Zielen. Wird nur im Treffermodus (Ziele begrenzt) genutzt. Standard: 15</German>
         </Key>
         <Key ID="STR_TAC_ShootingRange_PauseDurations">
             <English>Pause Durations</English>
+            <German>Pausenlänge</German>
         </Key>
         <Key ID="STR_TAC_ShootingRange_PauseDurationsDesc">
             <English>List of available pause durations during target pop-ups in seconds, separated by commas. Only used in Timed Mode. Default: 1, 2, 3, 4, 5</English>
+            <German>Liste der möglichen Pausen für aufklappende Ziele. Durch Kommas trennen. Wird nur im Zeitmodus genutzt. Standard: 1, 2, 3, 4, 5</German>
         </Key>
         <Key ID="STR_TAC_ShootingRange_DefaultPauseDuration">
             <English>Default Pause Duration</English>
+            <German>Standard Pausenlänge</German>
         </Key>
         <Key ID="STR_TAC_ShootingRange_DefaultPauseDurationDesc">
             <English>Default pause duration during target pop-ups in seconds. Default: 3</English>
+            <German>Standard Pausenlänge während Ziele aufklappen in Sekunden. Standard: 3</German>
         </Key>
         <Key ID="STR_TAC_ShootingRange_CountdownTimes">
             <English>Countdown Times</English>
+            <German>Countdown-Zeit</German>
```

Comment:
@System98 Can you verify this is in plural? (Same for `Durations`, `Pause Durations` and `Target Amounts`.) Because it's exactly the same as singular, and I have no idea how German works...